cypress/support/pages/course/pageEditCourse.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+class Edit_Course {
+  static #courseToEdit = '[aria-label="Kurs Mathe"]'
+  static #pageTitle = '[id="page-title"]'
+  static #dropDownCourse = '[class="btn btn-sm dropdown-toggle btn-course-dropdown"]'
+  static #btnCourseEdit = '[class="dropdown-item btn-course-edit"]'
+  static #selectClassesInput = '#classId_chosen > ul > li > input'
+  static #selectClassesDropdown = '#classId_chosen > div > ul > li:nth-child(1)'
+  static #selectClassesDeleteButtons = '#classId_chosen > ul > li > a'
+  static #selectClasses = '#classId_chosen > ul > li > span'
+  static #btnSubmit = '#main-content > section > form > div.modal-footer > button.btn.btn-primary.btn-submit'
+  static #classSearchString = 'ALLE'
+  static #numberStudentsInClass = 3
+  static #courseName = 'Mathe'
+  static #selectStudents = '#studentsId_chosen > ul > li'
+
+  clickOnCourse () {
+    cy.get(Edit_Course.#courseToEdit).click()
+  }
+
+  coursePageIsVisible () {
```

Comment:
here method name should start with _verb_ as per --> 'verb + noun'  in Camel Case --> showCoursePage() or goTo CoursePage